src/remote_prune

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python3 -u
+
```

Comment:
Let's define the format of the policy.yaml somewhere in this file? i.e. a nice description at the top of this file might be a good start